--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -403,13 +403,13 @@
   endif()
 endif()
 
-find_package(Python3 COMPONENTS Interpreter Development)
+find_package(Python3 COMPONENTS Interpreter)
 
 if(NOT Python3_Interpreter_FOUND)
   message(STATUS "python not found, using default python")
   unset(Python3_ROOT_DIR)
   unset(Python3_FIND_STRATEGY)
-  find_package(Python3 COMPONENTS Interpreter Development)
+  find_package(Python3 COMPONENTS Interpreter)
 endif()
 
 
